<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::TranslatorInspectorWidget</class>
 <widget class="QWidget" name="GammaRay::TranslatorInspectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="GammaRay::DeferredTreeView" name="translatorList">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="translationsSearchLine"/>
       </item>
       <item>
        <widget class="GammaRay::DeferredTreeView" name="translationsView">
         <property name="selectionMode">
          <enum>QAbstractItemView::ContiguousSelection</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetTranslationsButton">
         <property name="text">
          <string>Reset selected</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="languageChangeButton">
     <property name="text">
      <string>Send QEvent::LanguageChange</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::DeferredTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ui/deferredtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
